akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/HelloWorld.scala

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.akka.persistence.typed
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+import akka.persistence.typed.scaladsl.Effect
+import akka.persistence.typed.scaladsl.PersistentBehaviors
+
+/*
+Comparing a Behavior with a PersistentBehavior
+ */
+
+object HelloWorld {
+  final case class Greet(whom: String, replyTo: ActorRef[Greeting])
+  final case class Greeting(whom: String, numberOfPeople: Int)
+
+  private final case class KnownPeople(names: Set[String]) {
+    def add(name: String): KnownPeople = copy(names = names + name)
+    def numberOfPeople: Int = names.size
+  }
+
+  val behavior: Behavior[Greet] = greeter(KnownPeople(Set.empty))
+
+  private def greeter(people: KnownPeople): Behavior[Greet] =
+    Behaviors.receiveMessage { msg ⇒
+      msg.replyTo ! Greeting(msg.whom, people.numberOfPeople)
+      greeter(people.add(msg.whom))
+    }
+}
+
+object PersistentHelloWorld {
```

Comment:
This mapping is quite straightforward. If one understands what commands and events are, their semantics, and the function of onCommand/onEvent then I think the concepts map over well to the persistence version.